docs/security/principles/05-minimized-impact-of-security-breaches.mdx

```diff
@@ -0,0 +1,11 @@
+# P05 - Minimized impact of security breaches
+
+Even with robust security measures in place, user error or unforeseen vulnerabilities can lead to
+various security breaches, including the compromise of encryption keys or data leaks. Bitwarden
+should take available actions to help users limit the damage caused by such breaches, both in scope
+and duration. This involves:
```

Comment:
Is this intended to be comprehensive? If not, we might say "includes" instead

Reply:
Good point, fixed!